### ISSUE TITLE
Registration as photo editor with Android (action-edit)

### ISSUE THREAD

**Issue by BOFH2k** [k3b/LosslessJpgCrop#8]
When adding a photograph to a contact, the app searches for a photo editor intent (?) as provided by Google Photos or Simple Gallery. It would be nice if LJC could be used for that use case.

**Comment by k3b** [comment]
This is similar to #3

**Comment by k3b** [comment]
Implemented in v1.1.0.190518

**Comment by BOFH2k** [comment]
Nope, with 1.1.0.190518 the Android contact app still refuses to use llcrop:
Installieren Sie eine Fotobearbeitungs-App, um diesen Vorgang abzuschließen.

**Comment by k3b** [comment]
on my android-7.1 it works as expected: 

i can open contacts, select edit-contact and then select edit-photo which presents me llCrop.

To find out what is happening: 

* Can you install https://f-droid.org/de/packages/de.k3b.android.intentintercept ?
* Is intentintercept shown as photo-editor candidate?
* If yes can you send me the intend data action/data/mime that your contact app uses?

My contact app uses

* action PICK
* data null
* mime image/*

**Comment by BOFH2k** [comment]
#response_container_BBPPID{font-family: initial; font-size:initial; color: initial;}  Hmmm,on my Android 8.1 com.blackberry.contacts app 2.1902.1.16890 no intent seems to be called, as intentintercept doesn't appear...Thanks anyway!

**Comment by k3b** [comment]
May be this is an android-8.1 nad/or permission issue.

does your stock contacts app have any working photo editor? if yes we have the chance to find out what might be happening by inspecting the manifest of the working app.

On my android devices (7.0 and 7.1) https://f-droid.org/de/packages/com.simplemobiletools.contacts.pro is also working well.

**Comment by BOFH2k** [comment]
#response_container_BBPPID{font-family: initial; font-size:initial; color: initial;}  As I wrote initially: Google Photos or Simple Gallery (https://f-droid.org/app/com.simplemobiletools.gallery.pro) are picked up successfully. APhotoManager isn't, incidentally.

**Comment by k3b** [comment]
Ups: working with Simple Gallery but not with intentintercept the i have a source where i can go on investigating ....

**Comment by k3b** [comment]
@BOFH2k  in v1.1.1 I have added action com.android.camera.action.CROP (stolen from simple gallery). may be this fixes the issue

**Comment by k3b** [comment]
@BOFH2k is this issue now solved with  v1.1.1? can we close this ticket?

**Comment by BOFH2k** [comment]
Well, 1.1.2 is called by the BlackBerry contacts app but the cropped image is then saved (as per pop-up) and the contacts app doesn't seem to pick that path up...
